product_docs/docs/edbcloud/beta/using_cluster/06_analyze_with_superset.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Analyzing Your Data with Apache Superset"
+---
+You can use Apache Superset to analyze, explore, and visualize data stored in your PostgreSQL clusters.
+You can open Superset from the [EDB Cloud](https://portal.edbcloud.com/) portal using the **Analyze** links. 
+
+Default permissions include the ability to view data and create dashboards. To add or alter data sources, you need additional permissions. For more information on permissions to access Superset, see [Managing Superset Access](../administering_cluster/01a_superset_access).
+
+!!! Note
+    Apache Superset shares Single Sign On with EDB Cloud. If you log out of Superset you will be logged out of EDB Cloud and vice-versa.
+
+
+## Connecting Superset to your Cluster
+
+
+
+
+You need your host and port to connect to Superset. To find the host and port of your cluster:
+
+1. Sign in to the [EDB Cloud](https://portal.edbcloud.com/) portal.
+
+3. Go to the [Clusters](https://portal.edbcloud.com/clusters) page.
+4. Select the name of your cluster.
+5. Select the **Connect** tab and find **Host** and **Port**.
+
+
+To create a connection to an EDB Cloud cluster:
+
+1. Sign in to the [EDB Cloud](https://portal.edbcloud.com/) portal.
+
+2. Navigate to **Analyze > Connections**.
+3. Select **+ Database** to open the **Add database** dialog box.
+4. Enter a value for **DATABASE NAME**.
+5. Enter the connection string for your cluster in the **SQLALCHEMY URI** field, using the following format:
+
+    `postgresql://{username}:{password}@{host}:{port}/{database}?sslmode=require`
```

Comment:
`database` should be `dbname` here for consistency with PG.

Reply:
Passwords are not in plaintext, they're masked as below: 

<img width="742" alt="Screenshot 2021-09-20 at 9 55 06 PM" src="https://user-images.githubusercontent.com/70691901/134038431-1b4ab8c1-ef6a-436c-854f-f36b4f45cdd1.png">

Reply:
I don't mean in the UI, I mean on the backend. If an attacker compromises the superset database, e.g., do they get the passwords?

Reply:
Superset does not store password, AFAICT. It relies heavily on `libpq` to make a connection each time. 

Docs link:  https://superset.apache.org/docs/databases/postgres

Reply:
Something has to store the password, right? If the superset pods are all restarted, does the user need to re-authenticate all of the database connections?

Reply:
No, it does not ask user to re-add and authenticate database in case of pod restart. 

I'm guessing (and i could be wrong here), it does store these PQConninfo in cache key i.e. Redis server which is fairly just a `keys:values` directory.

Reply:
Is there any doc change needed related to this conversation?

Reply:
> I'm guessing (and i could be wrong here), it does store these PQConninfo in cache key i.e. Redis server which is fairly just a keys:values directory.

Can you confirm?

> Is there any doc change needed related to this conversation?

Yes, IMO we should have some sort of statement about how and where we securely store the credential, i.e., why you should trust us here.

Reply:
I added a note stating Superset encrypts the password.